CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 - The region key has been replaced by an i64 in the f64 version of rapier, increasing the range before panics occur.
 - Fix `BroadphaseMultiSap` not being able to serialize correctly with serde_json.
 
+### Added
+
+- `RigidBodySet` and `ColliderSet` have a new constructor `with_capacity`.
```

Comment:
Should this point to this PR?

Reply:
Ideally yes, but it's rare to have the PR link, I'm adding it arbitrarily on bigger/breaking/difficult PRs